ports/espressif/common-hal/mdns/Server.h

```diff
@@ -34,4 +34,5 @@ typedef struct {
     const char *instance_name;
     // "cpy-" "XXXXXX" "\0"
     char default_hostname[4 + 6 + 1];
```

Comment:
This is a total aside but I recently learned that in C you can use sizeof("char literal") in this context!
```c
typedef struct whatever {
    char default_hostname[sizeof("cpy-XXXXXX")];
} whatever;
```
this includes the terminating '\0' byte.

https://godbolt.org/z/hEvxYe1xq